explorer/app/components/Project/components/Section/components/SectionTitle/sectionTitle.tsx

```diff
@@ -3,15 +3,16 @@ import { Typography } from "@mui/material";
 import React from "react";
 
 interface Props {
+  className?: string;
   title: string;
 }
 
-export const SectionTitle = ({ title }: Props): JSX.Element => {
+export const SectionTitle = ({ className, title }: Props): JSX.Element => {
```

Comment:
Updated SectionTitle to take additional styles (margin bottom) for non collapsable sections.

Reply:
Thanks for the detail here, F! Is there a way we can get around suppling a class name as a prop? For example, can we push the knowledge of the margin bottom to where the collapsable functionality resides?

Reply:
The classname props is added for the benefit of styling a component... see https://emotion.sh/docs/styled#styling-any-component

Reply:
Thanks Fran!